src/test/java/hotel/models/ManagerTest.java

```diff
@@ -0,0 +1,114 @@
+package hotel.models;
+
+import customexceptions.InvalidHotelActionException;
+import hotel.commodities.*;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ManagerTest {
+	private static Hotel hotel;
+	private static Manager manager;
+
+	@BeforeAll
+	static void setUp(){
+		hotel = new Hotel("California", null);
+		manager = new Manager("Ivan");
+		manager.setHotel(hotel);
+	}
+
+	@AfterEach
+	void tearDown(){
+		hotel.getRooms().clear();
+		hotel.getTakenRoomNumbers().clear();
+		manager.getBookedRoomsNumbers().clear();
+	}
+
+	@Test
+	void bookThreeRooms() {
+		//given
+		Set<AbstractCommodity> firstRoomCommodities = new HashSet<>(Arrays.asList(new Bed(BedType.SINGLE), new Shower(), new Toilet()));
```

Comment:
Lines in general case are 120 characters long. This is used to make them read on every screen. Good practice is to split the row when becomes too long